blockchain/block/utils.go

```diff
@@ -47,29 +45,3 @@ func calculateTransferAmount(acts []action.SealedEnvelope) *big.Int {
 	}
 	return transferAmount
 }
-
-// VerifyBlock verifies the block signature and tx root
-func VerifyBlock(blk *Block) error {
-	// verify new block's signature is correct
-	if !blk.VerifySignature() {
-		return errors.Errorf(
-			"failed to verify block's signature with public key: %x",
-			blk.PublicKey(),
-		)
-	}
-
```

Comment:
we can still keep this func, just replace with bottom part with `b.VerifyTxRoot()`

Reply:
This func name is unclear. It doesn't verify blk entirely

Reply:
After discussion, it is renamed to `VerifySignatureAndTxRoot`

Reply:
can you explain why we need to create this function for block? It doesn't help at all.

Reply:
revert